conjure/outputs.go

```diff
@@ -50,16 +50,20 @@ func (f *OutputFile) Write() error {
 }
 
 func (f *OutputFile) Render() ([]byte, error) {
-	f.file.HeaderComment("This file was generated by Conjure and should not be manually edited.")
+	bytes, err := f.render()
+	return bytes, errors.Wrapf(err, "failed to render bytes for file %s", f.absPath)
```

Comment:
nit, but slight preference for the following:

```suggestion
	bytes, err := f.render()
	if err != nil {
	   return nil, errors.Wrapf(err, "failed to render bytes for file %s", f.absPath)
	}
	return bytes, nil
```

It is true that, functionally, this is almost identical to what you have, since `errors.Wrapf` returns `nil` if `err` is `nil`, and this construction is more verbose.

However, for myself, over time I've found that making this explicit reduces cognitive overhead, and when debugging it also makes it easier to set break points/be able to tell immediately whether the "success" or "failure" logic branch is being executed.

Reply:
2aa3e01950ec195a6f1487e7077ce5427f4cb791